src/T.tsx

```diff
@@ -11,35 +11,54 @@ export interface ITProps {
   isHTML?: boolean;
   domPurifyConfig?: IDomPurifyConfig;
   placeholders?: Array<string | number>;
+  context?: string;
 }
 
-export const T = (props: ITProps) => {
+export const percentageFix = (text: string): string => {
```

Comment:
@marcneander what do you think about renaming this to something more descriptive like `escapePercentage`?

Reply:
@mariakatsourani i like it